documentation/docs/02-layouts.md

```diff
@@ -80,11 +80,14 @@ If an error component has a [`load`](#loading) function, it will be called with
 
 ```html
 <script context="module">
+	/**
+	 * @type {import('@sveltejs/kit').ErrorLoad}
```

Comment:
It's a bit of a mix, but usually for 1-line javadocs we write them as:
```
/** @type {import('@sveltejs/kit').ErrorLoad} */
```

We should probably change the others that aren't condensed to 1-line for consistency

Reply:
changed this one